Library/Homebrew/dev-cmd/test-bot.rb

```diff
@@ -659,6 +659,7 @@ def homebrew
         test "brew", "tests", *tests_args
         test "brew", "tests", "--no-compat"
         test "brew", "readall", "--syntax"
+        test "brew", "official-cmd-tap-tests"
```

Comment:
Can you please add the file implementing this command to your commit? It's difficult to review and/or test otherwise. :wink: